calico_containers/pycalico/util.py

```diff
@@ -142,7 +142,7 @@ def validate_icmp_type(icmp_type):
     Validate that icmp_type is an integer between 0 and 255.
     If not return False.
 
-    :param icmp_type:
+    :param icmp_type: string or number value representing an icmp/icmpv6 type
```

Comment:
We have some tests in calicoctl where it is a stringified number like "100", but I guess that doesn't have to be explicit here.